### PR TITLE
Find and download the coverage profile file from the latest healthy build

### PR DESCRIPTION
**Pull request by steuhs**
A tool to find and download the coverage profile file from the latest healthy build stored in given gcs directory.

### CODE REVIEW

File: robots/coverage/downloader/downloader.go [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+	"strconv"
+
+	"cloud.google.com/go/storage"
+	"github.com/sirupsen/logrus"
+	"google.golang.org/api/iterator"
+)
+
+const (
+	//covProfileCompletionMarker is the file name of the completion marker for coverage profiling
+	covProfileCompletionMarker = "profile-completed"
```
[the remark this answers is not in the document]
Comment (steuhs) [reply robots/coverage/downloader/downloader.go]: Does Prow have any existing Json parser? @Katharine @cjwagner

[the remark this answers is not in the document]
Comment (cjwagner) [reply robots/coverage/downloader/downloader.go]: Just use the standard library: https://golang.org/pkg/encoding/json/

**Pull request merged**